src/WorkspaceContext.ts

```diff
@@ -136,6 +137,30 @@ export class WorkspaceContext implements vscode.Disposable {
                         }
                     });
             }
+            // on change of swift debugger type
+            if (event.affectsConfiguration("swift.debugger.useDebugAdapterFromToolchain")) {
+                if (configuration.debugger.useDebugAdapterFromToolchain) {
+                    if (!(await verifyDebugAdapterExists(this.toolchain))) {
```

Comment:
Might be nice to have a logged event to explain that the DAP was not found.

Reply:
This needs cleaned up a little. I can add a log event. Would be useful if I could disable settings under certain situations but VSCode doesn't seem to allow this.

Reply:
Added a log event